app/crypto/create-stx-tx.ts

```diff
@@ -0,0 +1,24 @@
+import BN from 'bn.js';
+import BigNumber from 'bignumber.js';
```

Comment:
Why both?

Reply:
I need the extra features, namely formatting and decimal precision. 

Initially I thought these were basically the same lib, but bn is just for manipulating integers.

Reply:
Yeah, both the explorer (old) and authenticator use `bignumber.js` for STX<->µSTX, since it has a better API for it. Kind of clunky, I know.

Reply:
I first used the Web internationalisation API for number formatting, but owing to having convert to `number` first, even that ran into imprecision issues.

Reply:
Think it'd be great if we could initialize `BN` internally in stacks-transaction-js.